app/policies/video_policy.rb

```diff
@@ -1,9 +1,15 @@
 class VideoPolicy < ApplicationPolicy
-  def new?
+  def create?
```

Comment:
Doesn't this change impact this controller action?

https://github.com/forem/forem/blob/2075a021fa15d3836943bbd281c819f5d4cbb393/app/controllers/videos_controller.rb#L6

Reply:
I'm not actually sure - I'll have to check. Thanks for picking that up, for now I'm going to push a method that encompasses 'enabled?' and leaves 'new?' as is. It will be more verbose which seems okay. Will re request a review in a bit.